src/govuk/components/cookie-banner/cookie-banner.yaml

```diff
@@ -15,52 +15,52 @@ params:
   type: object
   required: false
   description: The additional attributes that you want to add to the button or link. For example, data attributes.
-- name: banners
+- name: messages
   type: array
   required: true
-  description: Required. The different kinds of banners you can pass into the cookie banner. For example, the question banner and the replacement message.
+  description: Required. The different messages you can pass into the cookie banner. For example, the question message and the replacement message.
   params:
     - name: headingText
       type: string
       required: false
-      description: The heading text that displays in the cookie banner. You can use any string with this option. If you set `headingHtml`, `headingText` is ignored.
+      description: The heading text that displays in the cookie message. You can use any string with this option. If you set `headingHtml`, `headingText` is ignored.
     - name: headingHtml
       type: string
       required: false
-      description: The heading HTML to use within the cookie banner. You can use any string with this option. If you set `headingHtml`, `headingText` is ignored. If you are not passing HTML, use `headingText`.
+      description: The heading HTML to use within the cookie message. You can use any string with this option. If you set `headingHtml`, `headingText` is ignored. If you are not passing HTML, use `headingText`.
     - name: text
       type: string
       required: false
-      description: Required. The text for the main content within the banner. You can use any string with this option. If you set `html`, `text` is not required and is ignored.
+      description: Required. The text for the main content within the cookie message. You can use any string with this option. If you set `html`, `text` is not required and is ignored.
     - name: html
       type: string
       required: false
-      description: Required. The HTML for the main content within the banner. You can use any string with this option. If you set `html`, `text` is not required and is ignored. If you are not passing HTML, use `text`.
+      description: Required. The HTML for the main content within the cookie message. You can use any string with this option. If you set `html`, `text` is not required and is ignored. If you are not passing HTML, use `text`.
     - name: hidden
       type: boolean
       required: false
-      description: Defaults to false. If you set it to `true`, the banner is hidden. You can use `hidden` for client-side implementations where the replacement message HTML is present, but hidden on the page.
+      description: Defaults to false. If you set it to `true`, the message is hidden. You can use `hidden` for client-side implementations where the replacement message HTML is present, but hidden on the page.
```

Comment:
@EoinShaughnessy Technically this can be applied to any message, so I don't think we want to say that only the replacement message is hidden? We give it as an example use case, but a user may have a scenario where they use it on a different message and that would still work.

Reply:
@vanitabarrett That makes sense - thanks. Is there any chance a user, reading "the message is hidden," could think we mean the cookie message?

Reply:
@EoinShaughnessy Possibly, although I think the example should help with that. But I think it's ok for them to think we mean the cookie message if they have a valid reason for needing to hide the cookie message. For example: they might want to hide the cookie message initially and only show it with JavaScript so the banner only appears when JS is enabled, and I think that would be a valid thing for them to do.

Reply:
@vanitabarrett That's a great explanation - thanks Vanita. I reckon the original wording is fine in that case.